src/Components/Commodity/CommodityDetails.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable react/no-array-index-key */
+import { useEffect, useState } from 'react';
+import { useParams, Link } from 'react-router-dom';
+import Table from 'react-bootstrap/Table';
+import { Button } from 'react-bootstrap';
+import { UserKey } from '../ApiUrl/API';
+import { CommodityHistoryPriceUrl } from '../ApiUrl/ApiUrl';
+
+function CommodityDetailsPrice() {
+  const { symbol } = useParams();
+  const [commodityData, setCommodityData] = useState(null);
+
+  const fetchData = async () => {
+    const response = await fetch(`${CommodityHistoryPriceUrl}${symbol}${UserKey}`);
+    if (response.ok) {
+      try {
+        const data = await response.json();
+        setCommodityData(data);
+      } catch (error) {
+        console.error('Error fetching commodity details', error);
+      }
+    } else {
+      console.error('Error fetching commodity details', response.status);
```

Comment:
Good job working on the project requirements but i noticed you disabled eslint in this file it would be nice if you fixed this by enabling eslint because it is against microverse policy to disable linters because linters help us write clean code. Please kindly do this in all files were eslint was disabled